cmd/deploy/deploy.go

```diff
@@ -60,15 +61,16 @@ var tempKubeConfigTemplate = "%s/.okteto/kubeconfig-%s-%d"
 
 // Options options for deploy command
 type Options struct {
-	ManifestPath     string
-	Name             string
-	Namespace        string
-	K8sContext       string
-	Variables        []string
-	Manifest         *model.Manifest
-	Build            bool
-	Dependencies     bool
-	servicesToDeploy []string
+	OriginalManifestPath string
```

Comment:
I'm trying to think in a better name for this. `SpecifiedManifestPath` or something similar to indicate that is the one specified by the user when the command is executed. Not sure if `Original` could represent that

Reply:
ManifestPathFlag ? ¿

Reply:
Maybe splitting into two structs can do the trick. One to store flags/args information and another one to be the one used by the command. WDYT?

Reply:
🤔  i like the idea, but... afraid of the scope of the PR